rust_src/rustfmt.helper.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Add this to your Emacs config and it will format the file whenever you save.
+#
+#     (setq rust-format-on-save t)
+#
+# Then make sure this script is named `rustfmt` and in your path before ~/.cargo/bin.
```

Comment:
setting `LD_LIBRARY_PATH=$(rustc --print sysroot)/lib:$LD_LIBRARY_PATH` seems easier than these tweaks. I would rather suggest to add instructions to set `LD_LIBRARY_PATH` in readme.

Reply:
Why pollute that variable because of one busted command?

Reply:
No one is telling anyone to use it. This is offered as an example solution to the problem. Hopefully the compiler gets fixed and this can go away.